app/src/main/java/petermcneil/musiclibrary/controllers/RecordingController.java

```diff
@@ -1,48 +1,56 @@
 package petermcneil.musiclibrary.controllers;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
 import petermcneil.domain.Recording;
 import petermcneil.musiclibrary.services.RecordingService;
 
-@RestController
+@Controller
 public class RecordingController {
     private RecordingService db;
+    private static final Logger LOG = LoggerFactory.getLogger(RecordingService.class);
```

Comment:
Logger should be initialised for the same class its in, here it is using `RecordingService` not `RecordingController`